libslide/src/lib.rs

```diff
@@ -75,8 +76,11 @@
 //! Because of these limitations in string rules, slide also supports function rules. These are
 //! functions of the form
 //!
-//! ```ignore
+//! ```
+//! # use libslide::Expr;
+//! # type _A =
 //! fn(expr: Expr) -> Option<Expr>
+//! # ;
 //! ```
```

Comment:
doing this so that rustdoc accepts it as rust code and does not display it as "ignored". #s are added to tell rustdoc to ignore displaying those lines